### ISSUE TITLE
Release announcement should note incompatible devices

### ISSUE THREAD

**Issue by DirkSchmitt** [koreader/koreader#12794]
#### Issue

The last two releases of koreader are problematic on *Tolino Epos 1* and other devices. 

This is not visible on the release announcements and will bring inexperienced users in problematic situations. 

IMHO a blocking bug like  #12769 (2024.11) and #12210 (2024.07) should be added as a note to the release announcement.

**Comment by mergen3107** [comment]
Usually, in such cases there is a .1 release like 2024.11.1. 

@Frenzie ?

**Comment by DirkSchmitt** [comment]
> Usually, in such cases there is a .1 release like 2024.11.1.

This wasn't the case for the last 2 releases :-(

**Comment by pazos** [comment]
> The last two releases of koreader are problematic on Tolino Epos 1 and other devices.

Yup :) Use a nightly.

> This is not visible on the release announcements and will bring inexperienced users in problematic situations.

Sadly (or not) most of our users get updates via OTA updates, not using the github releases page. They won't see that message.

> Usually, in such cases there is a .1

If it is deemed relevent, in this case it wasn't (it was tagged for 2025.01) and I 110% agree.

It affects a few android-tolino devices, where the app is another APK and a crash on launch will just return you to your launcher. It doesn't deserve a point release at all.

In fact documenting these kind of breaks takes maintainer time and serves little pupose. But, even more importantly, frame the issue wrong:

The issue that happened was: **no tolino user tested a single nightly  in 2 weeks, so a bug made by a refactor sliped into a monthly release. 

From the pov of the devs/maintainers the bug was fixed within 24 from the report and the current released version of KOReader (which is always the nightly from the master branch) no longer has an issue.


**tl;dr:** not a big deal to handle by users, 99'9999999999% of users are unnafected (including many w/ tolinos), maintainer burden.

**Comment by Frenzie** [comment]
I have added a note to the release in case it's helpful for someone.

@pazos Btw, could you check Gitter when you have some time?

**Comment by Frenzie** [comment]
> Thanks – but the issue is more serious. The koreader app freeze and is unusable.

The system doesn't do anything even when pressing home or switching to another app? That's certainly unexpected. But no matter, it's fixed now.

**Comment by offset-torque** [comment]
> I know we have a pinned ticket in Issues on how to track nightlies.

I see that nightly versions can be useful (even necessary) for some users. I can add a section to the user guide under the **Troubleshooting** section which also includes links to relevant pages like **"How to track..."**. I always thought nightlies as a developer specific version so I assumed nightly users don't need things to be explained. But if non-technical users are directed to nightlies due to bugfixes, I think some documentation is necessary. What do our devs think?

**Comment by Frenzie** [comment]
You stick to / revert to the old version that works for you unless you want to help test.
